l10n/po/zh_CN/_guides/rest-json.adoc.po

```diff
@@ -621,7 +621,7 @@ msgstr "启动Quarkus："
 #: upstream/_guides/rest-json.adoc:323
 
 msgid "open a browser to `http://localhost:8080/fruits.html`"
-msgstr "打开浏览器访问 `http://localhost:8080/fruits.html` "
+msgstr "打开浏览器访问 `<a href=\"http://localhost:8080/fruits.html\" class=\"bare\">http://localhost:8080/fruits.html</a>` "
```

Comment:
I think it does not need to add `<a>` here.

Reply:
This <a> exists during machinetranslation.

Reply:
#89 

![image](https://user-images.githubusercontent.com/55195642/174520685-5439c5bb-35bd-4c64-80b8-d574f13b2c07.png)

Reply:
I delete the`<a>` at first commit, recovery `<a>` at second commit.

Reply:
sorry, it is my fault.

Reply:
The `<a>`  keep or delete?

Reply:
Deleted.